command/agent/host/windows.go

```diff
@@ -0,0 +1,58 @@
+// +build windows
+
+package host
+
+import (
+	"os"
+	"syscall"
+	"unsafe"
+)
+
+func uname() string {
+	return ""
+}
+
+func resolvConf() string {
+	return ""
+}
+
+func etcHosts() string {
+	return ""
+}
+
+func mountedPaths() (disks []string) {
+	for _, c := range "ABCDEFGHIJKLMNOPQRSTUVWXYZ" {
+		d := string(c) + ":\\"
+		_, err := os.Stat(d)
+		if err == nil {
+			disks = append(disks, d)
+		}
+	}
+	return disks
+}
+
+type df struct {
+	size  int64
+	avail int64
+}
+
+func makeDf(path string) (*df, error) {
+	h := syscall.MustLoadDLL("kernel32.dll")
```

Comment:
This makes me really nervous. Let's at least use `syscall.LoadDLL` and return an error instead of panicking an agent if this DLL can't be loaded. I have no idea what the failure conditions are for this sort of operation, and I would hate for some overzealous antivirus to block this operation and crash agents.

Reply:
Fixed